generic3g/tests/Test_ExtensionAction.pf

```diff
@@ -0,0 +1,20 @@
+#include "MAPL_TestErr.h"
+module Test_ExtensionAction
+   use mapl3g_ExtensionAction
+   use mapl3g_NullAction
+   use pfunit
+   use ESMF_TestMethod_mod
+   implicit none
+
+contains
+
+   @Test(type=ESMF_TestMethod, npes=[1])
+   subroutine test_runs_invalidate(this)
+      class(ESMF_TestMethod), intent(inout) :: this
+      type(NullAction) :: action
+
+      @assert_that(action%runs_invalidate(), is(false()))
+
+   end subroutine test_runs_invalidate
```

Comment:
Debatable that this is worthwhile as a test.   OTOH, certainly not worth deleting either.

Reply:
It may be overkill. I was checking that the default ExtensionAction.runs_invalidate() returns .FALSE. which skips the update_sources() and ExtensionAction.invalidate(). I added a similar test of AccumulatorAction.runs_invalidate() to make sure it would run.